src/unitxt/operators.py

```diff
@@ -71,32 +71,64 @@ class MapInstanceValues(StreamInstanceOperator):
         strict (bool): If True, the mapping is applied strictly. That means if a value
             does not exist in the mapper, it will raise a KeyError. If False, values
             that are not present in the mapper are kept as they are.
+        process_every_value (bool): If True, all fields to be mapped should be lists, and the mapping
+            is to be applied to their individual elements. If False, mapping is only applied to a field
+            containing a single value.
+
+    Examples: MapInstanceValues with mappers={"a": {"1": "hi", "2": "bye"}}, and process_every_value = True,
```

Comment:
@dafnapension @elronbandel - I think we should agree on the example format for all operators.  I think we should aim that each example will be self contained, and one could cut and past it.  Fro example:

Replace '1' with 'hi' and '2' with 'bye' in field 'a' of all streams:
`MapInstanceValues(mappers={"a": {"1": "hi", "2": "bye"}})`
So {"a":1 , "b": 2} becomes "a":"hi" , "b": 2} 

Assuming 'a' is field with a list of values, replace  all 1s with 'hi' and all 2s with 'bye'
`MapInstanceValues(mappers={"a": {"1": "hi", "2": "bye"}}, process_every_element=True`
 So {"a": [1, 2], "b": 2} becomes {"a": ["hi", "bye"], "b": 2}.

 Ensure all values are mapped in the instance
 `MapInstanceValues(mappers={"a": {"1": "hi", "2": "bye"}}, strict=True)`
 So if the input is  {"a":3 , "b": 2} it will throw an exception, since the mapping of 3 in not in the mapper.

Reply:
I accept.